src/js/htmlSanitizer.js

```diff
@@ -82,6 +87,24 @@ function leaveOnlyWhitelistAttribute($html) {
   });
 }
 
+/**
+ * Remove invalid attribute values
+ * @private
+ * @param {jQuery} $html jQuery instance
+ */
+function removeInvalidAttributeValues($html) {
+  $html.find('*').each((index, node) => {
```

Comment:
I think it's unreasonable to navigate all nodes. You can only handle tags that have property values ​​that can cause XSS.